<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quitter">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Identité</string>
    </property>
    <widget class="QLineEdit" name="Nom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="Prenom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Maj">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Mise à jour</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Décrémenter le porte monnaie</string>
    </property>
    <widget class="QPushButton" name="Payer">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Payer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre d'unités</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre d'unités à décrémenter</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nb_unite_spin_dec">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>44</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nb_unite">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Incrémenter le porte monnaie</string>
    </property>
    <widget class="QPushButton" name="Charger">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nb_unite_spin_inc">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>44</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre d'unités à incrémenter</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Disconnect">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>400</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_carte">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sélectionner la carte </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Debug</string>
    </property>
    <widget class="QTextEdit" name="Saisie_Affichage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="Affichage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="image_mathilde">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/téléchargement.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image_marine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/Downloads/marine.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>Connect</zorder>
   <zorder>Quitter</zorder>
   <zorder>Disconnect</zorder>
   <zorder>Select_carte</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>Affichage</zorder>
   <zorder>image_mathilde</zorder>
   <zorder>image_marine</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
